2-collectors/scala-stream-collector/pubsub/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/PubSubCollector.scala

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2013-2017 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0, and
+ * you may not use this file except in compliance with the Apache License
+ * Version 2.0.  You may obtain a copy of the Apache License Version 2.0 at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the Apache License Version 2.0 is distributed on an "AS
+ * IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the Apache License Version 2.0 for the specific language
+ * governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.snowplow
+package collectors
+package scalastream
+
+import scalaz._
+import Scalaz._
+
+import model._
+import sinks.PubSubSink
+
+object PubSubCollector extends App with Collector {
```

Comment:
I'd advise against `App` trait in any application https://www.reddit.com/r/scala/comments/4zvelw/app_trait_vs_main_method/